src/iarc7_motion/iarc_tasks/go_to_roomba_task.py

```diff
@@ -0,0 +1,190 @@
+import math
+import rospy
+import tf2_ros
+import tf2_geometry_msgs
+import threading
+
+from geometry_msgs.msg import TwistStamped
+from geometry_msgs.msg import Point
+from geometry_msgs.msg import PointStamped
+
+from task_utilities.acceleration_limiter import AccelerationLimiter
+
+from .abstract_task import AbstractTask
+from iarc_tasks.task_states import (TaskRunning,
+                                    TaskDone,
+                                    TaskCanceled,
+                                    TaskAborted,
+                                    TaskFailed)
+from iarc_tasks.task_commands import (VelocityCommand, NopCommand)
+
+from translate_stop_planner import TranslateStopPlanner
+
+class GoToRoombaState(object):
+    init = 0
+    translate = 1
+
+class GoToRoombaTask(AbstractTask):
+
+    def __init__(self, task_request):
+        super(GoToRoombaTask, self).__init__()
+
+        self._roomba_id = task_request.frame_id + '/base_link'
+
+        if self._roomba_id is None:
+            raise ValueError('A null roomba id was provided')
+
+        self._roomba_odometry = None
+        self._roomba_point = None
+        self._limiter = AccelerationLimiter()
+
+        self._canceled = False;
+        self._last_update_time = None
+        self._current_velocity = None
+        self._transition = None
+
+        self._lock = threading.RLock()
+
+        self._x_position = task_request.x_position
```

Comment:
The task request wouldn’t be sending in a x or y position, but rather a roomba id, so these are going to be 0.0. This won’t  impact the translate stop planner, but it’s unnecessary code. You can just pass in None for the x and y initial positions to the translate stop planner and that will also work.

Reply:
Changed the values to None